app/views/menus/preview.html.erb

```diff
@@ -0,0 +1,41 @@
+<div>
+  <%= image_tag "kale.png", :class => "top-image" %>
+</div>
+
+<h1 class="preview-menu-name"><%= @menu.name %></h1>
+
+<% @menu.menu_sections.reverse.each do |s| %>
```

Comment:
Using `.reverse` in the view isn't excessive, but it would be better fit in the controller, and `.reverse` is a ruby method, while a sql solution is going to be more performant. 

```ruby
@menu.menu_sections.order("created_at desc")
```